src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
```

Comment:
Please add a JavaDoc comment indicating the intention of this method. Currently, it seems if the **first** field of entry contains something, it returns true.